README.md

```diff
@@ -1,2 +1,101 @@
 # dart_multihash
-A dart-lang implementation of the Multihash protocol
+
+This is an implementation of the 
+[Multihash](https://github.com/multiformats/multihash)
+standard in Dart.
+This package provides an extensible representation of cryptographic hashes.
+
+# Install
+
+To install `dart_multihash`, simply run the following command.
+
+```elixir
+flutter pub add dart_multihash
```

Comment:
Should the installation instructions be fore `Dart` first and _then_ for `Flutter`? 💭 
Or are we expecting that this will _only_ be used in `Flutter` Apps and _never_ anywhere `else`?

Reply:
Fixed on https://github.com/dwyl/dart_multihash/pull/2/commits/15bcf43e1667b8c6c79bbae7f545085f229f8af6